package.json

```diff
@@ -461,16 +461,22 @@
   },
   "dependencies": {
     "browser-stdout": "1.3.1",
+    "chalk": "^2.4.1",
     "commander": "2.19.0",
     "debug": "3.1.0",
     "diff": "3.5.0",
     "escape-string-regexp": "1.0.5",
+    "findup-sync": "^2.0.0",
     "glob": "7.1.2",
     "growl": "1.10.5",
     "he": "1.1.1",
+    "js-yaml": "^3.12.0",
+    "log-symbols": "^2.2.0",
     "minimatch": "3.0.4",
     "mkdirp": "0.5.1",
-    "supports-color": "5.4.0"
+    "strip-json-comments": "^2.0.1",
```

Comment:
Since you're adding dependencies anyway, how about we drop our out-of-date internal version of `ms` and use the npm package instead?

Reply:
seems tangential, but a good idea.  see #3557